src/client/components/LocalizedImage.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react'
+import styled from '@emotion/styled'
+import { useCurrentLocale } from 'components/utils/CurrentLocale'
+
+type Props = React.ComponentPropsWithoutRef<'img'> & {
+  src: string
+}
+
+const Image = styled.img`
+  max-width: 100%;
+`
+
+const getLocaleSuffix = (currentLocale: string) => {
+  switch (currentLocale) {
+    case 'se':
+      return 'se'
+    case 'no':
+      return 'no'
+    case 'dk':
+      return 'dk'
+    default:
+      return 'en'
+  }
+}
+
+const getLocaleSrc = (basePath: string, currentLocale: string) => {
+  const pathParts = basePath.split('.')
+  const localePath = [
+    ...pathParts.slice(0, -1),
+    getLocaleSuffix(currentLocale),
+    pathParts.slice(-1),
+  ].join('.')
```

Comment:
Looks good! Just details, but do we want to use a dot or a dash? I was thinking dash, but maybe dot is easier to spot :)

Reply:
Hmm yeah I haven't really got a strong opinion I guess. Just that we already use "dash" to separate words so I thought this would be clearer.